mappings/net/minecraft/world/PersistentStateManager.mapping

```diff
@@ -8,10 +8,20 @@ CLASS net/minecraft/class_26 net/minecraft/world/PersistentStateManager
 		ARG 2 dataFixer
 	METHOD method_120 get (Ljava/util/function/Supplier;Ljava/lang/String;)Lnet/minecraft/class_18;
 		ARG 1 factory
-		ARG 2 id
+		ARG 2 fileName
 	METHOD method_123 set (Lnet/minecraft/class_18;)V
+		ARG 1 persistentState
 	METHOD method_125 save ()V
+	METHOD method_17921 isCompressedNbt (Ljava/io/PushbackInputStream;)Z
+		ARG 1 pushbackInputStream
```

Comment:
This is not very descriptive. Maybe `stateData` ?

Reply:
what do you mean?

Reply:
The argument name, `pushbackInputStream`, is just the name of the class so it does not add any information.

Reply:
I'd just call this `in` (or `stream`, or `inoutStream`). There's no need for the parameter name to specify which type of stream it is...

Reply:
This push back input stream, as far as I know, is used to detect if the input dat file is in gzip compressed nbt format or plain nbt format